contributors/design-proposals/kubelet-eviction.md

```diff
@@ -425,6 +425,22 @@ from placing new best effort pods on the node since they will be rejected by the
 On the other hand, the `DiskPressure` condition if true should dissuade the scheduler from
 placing **any** new pods on the node since they will be rejected by the `kubelet` in admission.
 
+## Enforcing Node Allocatable
+
+To enforce [Node Allocatable](./node-allocatable.md), Kubelet primarily uses cgroups.
+However `storage` cannot be enforced using cgroups.
+
+Once Kubelet supports `storage` as an `Allocatable` resource, Kubelet will perform evictions whenever the total storage usage by pods exceed node allocatable.
+
+The trigger threshold for storage evictions will not be user configurable for the purposes of `Allocatable`.
+Kubelet will evict pods once the `storage` usage is greater than or equal to `98%` of `Allocatable`.
```

Comment:
I also think we should consider reusing the minimum reclaim value here.  In theory, the value should be configured to reduce hitting the same threshold in quick succession.  In my estimation, this should be the same, regardless of which threshold is being hit.  Under what circumstances would anyone want the allocation-min-reclaim to be different from eviction-min-reclaim?